recipes/approvaltests.cpp/7.0.0/conanfile.py

```diff
@@ -0,0 +1,59 @@
+import os
+from conans import ConanFile, tools
+
+
+class ApprovalTestsCppConan(ConanFile):
+    name = "approvaltests.cpp"
+    version = "7.0.0"
+    url = "https://github.com/conan-io/conan-center-index"
+    homepage = "https://github.com/approvals/ApprovalTests.cpp"
+    license = "Apache-2.0"
+    description = "Approval Tests allow you to verify a chunk of output " \
+                  "(such as a file) in one operation as opposed to writing " \
+                  "test assertions for each element."
+    topics = ("conan", "testing", "unit-testing", "header-only")
+    options = {"test_framework": ["catch2", "gtest", "doctest", "boost"]}
+    default_options = {"test_framework": "catch2"}
+    exports_sources = "patches/*"
+    no_copy_source = False
```

Comment:
```suggestion
    no_copy_source = True
```

It should be True, since is a header-only recipe